resources/config.rb

```diff
@@ -0,0 +1,66 @@
+unified_mode true
+
+property :fpm_service, String,
+  default: lazy { php_fpm_service },
+  description: 'Name of the PHP-FPM service to manage.
+Defaults to platform specific names, see libraries/helpers.rb'
+
+property :fpm_ini_control, [true, false],
+  default: lazy { php_fpm_ini_control },
+  description: 'Whether to manage the PHP-FPM configuration file or not.
+Defaults to false'
+
+property :fpm_conf_dir, String,
+  default: lazy { php_fpm_conf_dir },
+  description: 'Directory path of the PHP-FPM configuration file.
+Defaults to platform specific paths, see libraries/helpers.rb'
+
+property :ini_template, String,
+  default: lazy { php_ini_template },
+  description: 'Source template file to use for the PHP configuration file.
+Defaults to platform specific tempaltes, see libraries/helpers.rb'
+
+property :ini_cookbook, String,
+  default: lazy { php_ini_cookbook },
+  description: 'Cookbook to use for the PHP configuration file template.
+Defaults to the cookbook specified in libraries/helpers.rb'
+
+property :directives, Array,
+  default: lazy { php_directives },
+  description: 'Array of custom PHP directives to configure in the PHP configuration file.
+Defaults to the values set in libraries/helpers.rb'
```

Comment:
You use `Array` here but it's actually a `Hash`. Looking at how you use, I think you really mean a `Hash`. Please do the following:
- Change this to `Hash`
- Remove `php_directives` method and just replace it with a default of `{}`